### PR TITLE
839 implement the who we are page

### LINKED ISSUE #839: [Redesign] Implement the "Хто ми" (Who We Are) Page
### User Story 8: Implement the "Хто ми" (Who We Are) Page

**As a** visitor to the CraftMerge platform,  
**I want** to access detailed information about CraftMerge's mission, goals, and purpose,  
**So that** I can understand the platform's value and decide whether it aligns with my needs.

---

### Acceptance Criteria:

1. **Page Structure:**
   - **Header Section**:
     - Includes CraftMerge logo and navigation options ("Про нас," "Підприємства та сектори").
     - A search bar is available for easy navigation.
     - Buttons to **log in** and **register** are displayed.
   
2. **Content Section:**
   - **Page Title**: "Хто ми" (Who We Are) should be clearly visible at the top of the page.
   - **Introduction Paragraph**:
     - Describes CraftMerge as a **leading forum** in Western Ukraine in collaboration with **Lviv Polytechnic National University**.
   - **Mission Statement**:
     - Highlights the goal of **uniting Ukrainian producers and startups** and **expanding industry perspectives**.
   - **Platform Overview**:
     - Explains CraftMerge as a **communication hub** for experience exchange and discussion of trends and technologies.
     - States the **benefits** of the forum, such as learning innovative solutions, increasing visibility, and growing businesses.
   - **Image Section**:
     - Displays a relevant image (e.g., a presentation or a forum event) next to the text for visual engagement.

3. **Footer Section:**

4. **Design and Layout:**
   - The page has a **light background** with easy-to-read text.
   - The **text is aligned left** for better readability.
   - The image is **right-aligned** and proportional to the content section.

5. **Interactivity:**
   - **Links in the header and footer** should redirect users to the appropriate pages.
   - **CRAFTMERGE** should be **clickable** and redirect users to the main page
   - The contact details should be **clickable** for quick interaction (open email client or initiate a call).
   
6. **Responsive Design:**
   - The page layout adjusts for **desktop, tablet, and mobile** devices.
   - On smaller screens, the image moves **below the text** to maintain readability.

7. **Performance and SEO:**
   - Use **optimized images** to ensure fast page loading.
   - The page title and meta description should reflect the purpose and content of the "Хто ми" section for better SEO.

---

Mockup

image

---

Epic #831

### PR DESCRIPTION
**Pull request by parsival2022**
image

image

image

### CODE REVIEW

**Review by romanmyko — APPROVED** [review]

File: FrontEnd/src/pages/AboutUsPage/AboutUs.module.css [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+        width: 50%;
+        padding: 0 16px;
+    }
+
+    .about-us-section-content {
+        padding: 40px 104px 48px;
+    }
+
+    .about-us-section-content__header {
+        margin-top: 80px;
```
Comment (Lvyshnevska) [inline FrontEnd/src/pages/AboutUsPage/AboutUs.module.css]: There is a redundant margin-top that doubles the 80px spacing. The parent class .about-us-section-main already has padding: 80px, which is enough.

File: FrontEnd/src/pages/AboutUsPage/AboutUs.module.css [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+        padding: 0 16px;
+    }
+
+    .about-us-section-content__text {
+        width: 50%;
+        padding: 0 16px;
+    }
+
+    .about-us-section-content {
+        padding: 40px 104px 48px;
```
Comment (Lvyshnevska) [inline FrontEnd/src/pages/AboutUsPage/AboutUs.module.css]: There are redundant paddings. The parent class .about-us-section-main already provides the necessary spacing through its padding and gap properties.

File: FrontEnd/src/pages/AboutUsPage/AboutUs.module.css [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+    gap: 24px;
+}
+
+.about-us-section-content__header {
+    color: var(--about-us-header-color);
+    margin-bottom: 24px;
+    margin-top: 40px;
```
Comment (Lvyshnevska) [inline FrontEnd/src/pages/AboutUsPage/AboutUs.module.css]: There are redundant margins. The necessary spacing is provided through the padding and gap in .about-us-section-main class.

File: FrontEnd/src/pages/AboutUsPage/AboutUs.module.css [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+    padding-bottom: 32px;
+}
+
+.about-us-section-content__image-container{
+    width: 345px; 
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    overflow: hidden; 
+    padding-bottom: 40px;
```
Comment (Lvyshnevska) [inline FrontEnd/src/pages/AboutUsPage/AboutUs.module.css]: There is redundant padding-bottom. The necessary spacing is provided through the padding in .about-us-section-main class.

File: FrontEnd/src/pages/AboutUsPage/AboutUs.module.css [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+    width: 100%;
+}
+
+.about-us-section-content {
+    box-sizing: border-box;
+    min-width: 345px;
+    width: 375px;
+    padding: 0 15px;
+    display: flex;
+    justify-content: space-between;
```
Comment (Lvyshnevska) [inline FrontEnd/src/pages/AboutUsPage/AboutUs.module.css]: There is a redundant justify-content property. It has no effect on mobile and tablet screens. And it is only applied for screens > 1200px in media query.

File: FrontEnd/src/pages/AboutUsPage/AboutUs.module.css [hunk 6]
```diff
@@ -0,0 +1,126 @@
+
+.about-us-section {
+    min-width: 100vw;
+    background: var(--about-us-background-color);
+    justify-content: center; 
+    align-items: center;
```
Comment (Lvyshnevska) [inline FrontEnd/src/pages/AboutUsPage/AboutUs.module.css]: There are redundant justify-content and align-items properties that have no effect.

**Review by Lvyshnevska — APPROVED** [review]

**Pull request merged**